pkg/etcdenvvar/etcd_env.go

```diff
@@ -49,10 +51,15 @@ var envVarFns = []envVarFunc{
 //   NODE_%s_ETCD_URL_HOST
 //   NODE_%s_ETCD_NAME
 func getEtcdEnvVars(envVarContext envVarContext) (map[string]string, error) {
+	expectedEtcdClusterSize, err := operatorhelpers.GetExpectedEtcdSize(envVarContext.kubeSystemConfigMapLister)
```

Comment:
with this, we don't need to make an exception for libvirt. We always get the expected cluster size to compare with